dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java

```diff
@@ -622,7 +623,66 @@ public final Response deleteAction(@Context final HttpServletRequest request,
 
         return response;
     } // deleteAction
+    
+    /**
+     * Change the order of the steps in a scheme
+     * @param request                           HttpServletRequest
+     * @param workflowReorderActionStepForm     WorkflowReorderBean
+     * @return Response
+     */
+    @PUT
+    @Path("/reorder/step/{stepId}/order/{order}")
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public final Response reorderStep(@Context final HttpServletRequest request,
+                                        @PathParam("stepId")   final String stepId, 
+                                        @PathParam("order")    final int order) {
+
+        final InitDataObject initDataObject = this.webResource.init
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "initDataObject". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "initDataObject" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)